universal-application-tool-0.0.1/test/controllers/applicant/ApplicantProgramsControllerTest.java

```diff
@@ -42,19 +43,21 @@ public void index_withNoPrograms_returnsEmptyResult() {
 
   @Test
   public void index_withPrograms_returnsAllPrograms() {
```

Comment:
Can you rename this test to say something like "returns all relevant programs" or "returns all active and previous programs". It def doesn't return "all programs" now.

Reply:
Done!